playbooks/roles/nomad/meta/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+
+dependencies: []
```

Comment:
We don't need this file if it's empty anyway.  It was only required for Ansible Galaxy.

Reply:
Yep, will delete when I start polishing the PR(s).